.travis.yml

```diff
@@ -52,6 +54,7 @@ before_install:
   - export MVN_HOME=$HOME/apache-maven-3.6.3
   - if [ ! -d $MVN_HOME/bin ]; then wget https://archive.apache.org/dist/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.tar.gz -P $HOME; tar xzvf $HOME/apache-maven-3.6.3-bin.tar.gz -C $HOME; fi
   - export PATH=$MVN_HOME/bin:$PATH
+  - f=$(which javac); while [[ -L $f ]]; do f=$(readlink $f);done; export JAVA_HOME=${f%/bin/*}
```

Comment:
Can you please elaborate why this is needed? Thanks

Reply:
Hi Ethanlm,
Thanks for your review.
There is  an existed problem of Travis CI "JAVA_HOME is not exported for arm64 architecture", see:
https://travis-ci.community/t/java-home-is-not-exported-for-arm64-architecture/6993/3
in the above issue, there is an workaround mentioned, which is this line of code.

Reply:
Thanks for the detailed explanation!